frontend/src/pages/LocationsPage.js

```diff
@@ -3,10 +3,10 @@ import LocationList from "../components/LocationList";
 import {useState} from "react";
 import LocationMap from "../components/LocationMap";
 
-export default function LocationsPage() {
+export default function LocationsPage({geoLocation}) {
 
-    const locations = useLocations()
     const [mapIsEnabled, setMapIsEnabled] = useState(false)
+    const locations = useLocations(geoLocation)
 
     const switchView = (event, value) => {
         event.preventDefault()
```

Comment:
Afaik event is a click event, which default behaviour do you need to suppress?

Just from reading the function signature it is not clear what value is. When I see the call `switchView(event, false)` in `LocationMap` I can not tell what this function does. The mapping true = map and false = list is not transparent to the caller of the function.

Maybe refactor to a string constant that says what is switched to what, or better refactor to a simpler `toggleView` function so the different views do not have to care about the value argument?